src/Bundle/FrameworkBundle/src/DependencyInjection/Compiler/AddCacheWarmerPass.js

```diff
@@ -0,0 +1,25 @@
+const CompilerPassInterface = Jymfony.Component.DependencyInjection.CompilerPassInterface;
+const PriorityTaggedServiceTrait = Jymfony.Component.DependencyInjection.PriorityTaggedServiceTrait;
+
+/**
+ * @memberOf Jymfony.FrameworkBundle.DependencyInjection
+ */
+class AddCacheWarmerPass extends mix(CompilerPassInterface, PriorityTaggedServiceTrait) {
+    /**
+     *
+     * @param {Jymfony.Component.DependencyInjection.ContainerBuilder} container
+     */
+    process(container) {
+        if (! container.hasDefinition('cache_warmer')) {
+            return;
+        }
+
+        let warmers = this.findAndSortTaggedServices('kernel.cache_warmer', container);
+
+        if (warmers.isEmpty()) {
```

Comment:
`findAndSortTaggedServices` is a generator. You need to convert it to an array with `Array.from` to perform this check (and to pass it to the cache_warmer service definition)